cmd/cli/install.go

```diff
@@ -201,16 +203,19 @@ func newInstallCmd(config *helm.Configuration, out io.Writer) *cobra.Command {
 }
 
 func (i *installCmd) run(config *helm.Configuration) error {
+	fmt.Print("Validating command options")
```

Comment:
Previously I meant to say this is not really necessary apart from debugging. Could you remove this?

Reply:
I felt the install command was too quiet :)

Reply:
Let's resolve that as a part of https://github.com/openservicemesh/osm/issues/912 instead.